pom.xml

```diff
@@ -68,99 +68,45 @@
     </repositories>
```

Comment:
BTW weird that `<pluginRepositories>` is in a completely different part of the file.

Also recommend updating to HTTPS protocol.